.github/workflows/repo--release-please.yml

```diff
@@ -13,6 +13,7 @@ jobs:
   release-please:
     runs-on: [taiko-runner]
     steps:
-      - uses: google-github-actions/release-please-action@v4
+      - uses: googleapis/release-please-action@v4
         with:
           token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
+          config-file: release-please-config.json
```

Comment:
Do we need this or it already finds it by default?

Reply:
can remove, will push